### ISSUE TITLE
[Feature Request] Add edit files without any layout setting

### ISSUE THREAD

**Issue by kud** [math2001/FileManager#45]
"edit to the left" and "edit to the right" are good but it could be great to add a function to just open files without setting a layout.

**Comment by math2001** [comment]
I'm not sure I completely understand what you mean... 🙁 How is this behavior different from double clicking on the file name in the sidebar? Maybe you mean that "edit to the left/right" shouldn't *resize* the layout if it already exists?

By the way, thanks for raising issues and contributing to this package :heart:

**Comment by kud** [comment]
Ha! Got it. I'm sorry I was clear.

My problem is you can't open multiple files in one click with Sublime Text. And your feature is interesting because you can open multiple files via it. However I don't want to change my layout when I do that; that's why. :)

Am I clearer?

tl;dr: Add a way to open multiple files in one time.

**Comment by math2001** [comment]
Oh, I think I get you now 😄 

It shouldn't be too hard to do: make a special command lie `Open all` which would visible only if there are multiple files selected. Is that right?

**Comment by kud** [comment]
yes! that's it!
